cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/ExecutableBuilder.java

```diff
@@ -203,9 +206,30 @@ private Workflow compileWorkFlow(List<Map<String, Map<String, Object>>> workFlow
             String message = "Task: " + taskName + " syntax is illegal.\nBelow task name, there should be a map of values in the format:\ndo:\n\top_name:";
             try {
                 taskRawDataValue = taskRawData.values().iterator().next();
-                if (MapUtils.isNotEmpty(taskRawDataValue) && taskRawDataValue.containsKey(ScoreLangConstants.LOOP_KEY)) {
-                    message = "Task: " + taskName + " syntax is illegal.\nBelow the 'loop' keyword, there should be a map of values in the format:\nfor:\ndo:\n\top_name:";
-                    taskRawDataValue.putAll((Map<String, Object>) taskRawDataValue.remove(ScoreLangConstants.LOOP_KEY));
+                if (MapUtils.isNotEmpty(taskRawDataValue)) {
```

Comment:
I wrote it 'hacky' in the first place...
And this makes it way more complicate, maybe we should come with another way.
it does work anyways ;)